projects/vdk-core/src/vdk/internal/builtin_plugins/ingestion/ingester_router.py

```diff
@@ -148,7 +148,7 @@ def send_tabular_data_for_ingestion(
             )
         else:
             errors.report_and_throw(
-                UserCodeError(
+                VdkConfigurationError(
```

Comment:
What's the blamee listed as here for local runs?

Reply:
for local run it's user error, for cloud, platform. as other vdk configuration errors